logos/logos-ui/src/app/features/statistics/statistics.scss

```
@@ -152,6 +152,40 @@
   color: rgb(var(--color-typography-500));
 }
 
+// ── Provider-row badges (free VRAM / calibrating) ─────────────────────────────
+.stats-provider-vram-badge {
+  @include glass-surface;
+  border-radius: 9999px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)